persistence/src/test/resources/sql/candidate/candidate.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO candidates (name, email, phone, deleted)
```

Comment:
If you are using the default value of 'deleted', don't need to list it in the INSERT.